packages/esm-appointments-app/src/form/appointments-form.component.tsx

```diff
@@ -66,7 +68,7 @@ const appointmentsFormSchema = z.object({
   recurringPatternPeriod: z.number(),
   recurringPatternDaysOfWeek: z.array(z.string()),
   selectedDaysOfWeekText: z.string().optional(),
-  startTime: z.string(),
+  startTime: z.string().refine((value) => isValidTime(value)),
```

Comment:
This seems to be needed to prevent the form submission if the value fails validation